web/css/admin/admin-ui.styl

```diff
@@ -362,18 +345,14 @@ div.ui-state-droppable
 .users fieldset.option_groups
 	padding 0 0 0.6em 0
 
+.ui-accordion .ui-accordion-content.configuration_container
+	overflow visible
```

Comment:
I think we can remove this again now that we added `within: area` to the call to `position()`.